package.json

```diff
@@ -282,7 +282,7 @@
     "rollup-plugin-terser": "^7.0.2",
     "typescript": "~5.2.0",
     "uvu": "^0.5.6",
-    "wireit": "^0.14.0"
+    "wireit": "^0.9.5"
```

Comment:
Why downgrade wireit?

Reply:
Was wondering if that would fix, it did not